sdk/inc/azure/core/az_mqtt5_rpc_server_codec.h

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ *
+ * @brief Definition of #az_mqtt5_rpc_server_codec.
+ *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
+ */
+
+#ifndef _az_MQTT5_RPC_SERVER_CODEC_H
+#define _az_MQTT5_RPC_SERVER_CODEC_H
+
+#include <azure/core/az_result.h>
+#include <azure/core/az_span.h>
+
+#include <azure/core/_az_cfg_prefix.h>
+
+/**
+ * @brief MQTT5 RPC Server Codec options.
+ *
+ */
+typedef struct
+{
+  /**
+   * @brief Timeout in seconds for subscribing acknowledgement.
+   */
+  int32_t subscribe_timeout_in_seconds;
```

Comment:
Same as with client: timeouts should not exist in the codec layer since they cannot be used.
We can though have some recommendations (through `#define` statements) on what these timeouts should be by default.